### ISSUE TITLE
noobs does not allow setting proxies to access the internet

### ISSUE THREAD

**Issue by tps800** [raspberrypi/noobs#279]
NOOBS does not allow setting proxies to access internet. This might be necessary within various organisations like universities, collages, schools, companies, ...

Would therefore be quite nice if I could configure something like

ftp_proxy=http://myproxy.local:port
http_proxy=http://myproxy.local:port

to have NOOBS download OS-images from internet instead just telling me about "could not connect servers".

**Comment by generalh** [comment]
+1

**Comment by beta-tester** [comment]
maybe it is similar to
https://github.com/raspberrypi/noobs/issues/231
https://github.com/raspberrypi/noobs/pull/234

**Comment by lurch** [comment]
Nope, it doesn't have anything to do with #231 or #234 - those are concerned with accessing a *different* `os_list.json` whereas AFAIK this issue is concerned with accessing the 'normal' `os.json` when internet access is only available via an HTTP proxy, rather than directly.

**Comment by pradithya** [comment]
+1 encountered this issue while setting up windows 10 IoT core behind my company proxy.

**Comment by cccaballero** [comment]
+1
I just found this issue right now, It will be really useful to be able to set a proxy configuration.
